app/src/main/res/drawable-night/ic_volume_off.xml

```diff
@@ -4,7 +4,7 @@
     android:viewportWidth="24.0"
     android:viewportHeight="24.0">
     <path
-        android:fillColor="#FF000000"
+        android:fillColor="#FFFFFFFF"
```

Comment:
This fixed the issue for me, but I have no idea if this is the correct way.

Reply:
Yes, it is